framework/src/org/checkerframework/framework/type/AnnotatedTypeMirror.java

```diff
@@ -1966,6 +1963,24 @@ private void fixupBoundAnnotations() {
             }
         }
 
+        void setTypeVariable(TypeParameterElement typeParameterElement) {
```

Comment:
Explicit document why this method should have default visibility, same for overload below.